styles/pythonanywhere/catppuccin.user.css

```diff
@@ -0,0 +1,1424 @@
+      );
+      content: url("data:image/svg+xml;utf8,@{svg}");
+    }
```

Comment:
Line 1356 should have this themed already right?

```suggestion
```

Reply:
oops - you're totally right. its actually the other one that should be removed though, I fixed that in the commit